tests/wpt/mozilla/tests/mozilla/webvr/reftests/webvr-baseline.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Add the `reftest-wait` class to this, and remove it once rAF is done executing. This will wait until we've actually rendered things to take the reftest screenshot.